src/20241007_Multi_GHOStewardV2Upgrade/AaveV3Ethereum_GHOStewardV2Upgrade_20241007.t.sol

```diff
@@ -0,0 +1,450 @@
+        proposal.OLD_STEWARD()
+      ),
+      false
+    );
+
+    // Gho Bucket Steward
+    assertEq(
+      IGhoToken(AaveV3EthereumAssets.GHO_UNDERLYING).hasRole(
+        IGhoToken(AaveV3EthereumAssets.GHO_UNDERLYING).BUCKET_MANAGER_ROLE(),
+        proposal.GHO_BUCKET_STEWARD()
+      ),
+      true
+    );
+
+    assertTrue(
+      IGhoBucketSteward(proposal.GHO_BUCKET_STEWARD()).isControlledFacilitator(
+        AaveV3EthereumAssets.GHO_A_TOKEN
+      )
+    );
+    assertTrue(
+      IGhoBucketSteward(proposal.GHO_BUCKET_STEWARD()).isControlledFacilitator(
+        MiscEthereum.GSM_USDC
+      )
+    );
+    assertTrue(
+      IGhoBucketSteward(proposal.GHO_BUCKET_STEWARD()).isControlledFacilitator(
+        MiscEthereum.GSM_USDT
+      )
+    );
+
+    // Gho Aave Steward
+    assertEq(
+      IAccessControl(address(AaveV3Ethereum.ACL_MANAGER)).hasRole(
+        IACLManager(address(AaveV3Ethereum.ACL_MANAGER)).RISK_ADMIN_ROLE(),
+        proposal.GHO_AAVE_STEWARD()
+      ),
+      true
+    );
+
+    assertEq(
+      IUpgradeableLockReleaseTokenPool(MiscEthereum.GHO_CCIP_TOKEN_POOL).getBridgeLimitAdmin(),
+      proposal.GHO_CCIP_STEWARD()
+    );
+    assertEq(
+      IUpgradeableLockReleaseTokenPool(MiscEthereum.GHO_CCIP_TOKEN_POOL).getRateLimitAdmin(),
+      proposal.GHO_CCIP_STEWARD()
+    );
+
+    // GHO GSM Steward
+    assertEq(
+      IGsm(MiscEthereum.GSM_USDC).hasRole(
+        IGsm(MiscEthereum.GSM_USDC).CONFIGURATOR_ROLE(),
+        proposal.GHO_GSM_STEWARD()
+      ),
+      true
+    );
+    assertEq(
+      IGsm(MiscEthereum.GSM_USDT).hasRole(
+        IGsm(MiscEthereum.GSM_USDT).CONFIGURATOR_ROLE(),
+        proposal.GHO_GSM_STEWARD()
+      ),
+      true
+    );
+  }
+
+  function test_ghoAaveSteward_updateGhoBorrowCap() public {
+    executePayload(vm, address(proposal));
+
+    uint256 currentBorrowCap = _getGhoBorrowCap();
+    uint256 newBorrowCap = currentBorrowCap + 1;
+    vm.startPrank(RISK_COUNCIL);
+    IGhoAaveSteward(proposal.GHO_AAVE_STEWARD()).updateGhoBorrowCap(newBorrowCap);
+    assertEq(_getGhoBorrowCap(), newBorrowCap);
+  }
+
+  function test_ghoAaveSteward_updateGhoSupplyCap() public {
+    // Current config value is: 181338872942194741560446170431488
+    // existingConfig |= (10 << 116); turns into 830948836238514615306439858845646848
+    // this gives a value of 10 for the current supplyCap
+    uint256 configValue = 830948836238514615306439858845646848;
+    vm.mockCall(
+      address(AaveV3Ethereum.POOL),
+      abi.encodeWithSelector(AaveV3Ethereum.POOL.getConfiguration.selector),
+      abi.encode(configValue)
+    );
+
+    uint256 currentSupplyCap = _getGhoSupplyCap();
+    assertEq(currentSupplyCap, 10);
+    uint256 newSupplyCap = currentSupplyCap + 1;
+
+    executePayload(vm, address(proposal));
+
+    IGhoAaveSteward steward = IGhoAaveSteward(proposal.GHO_AAVE_STEWARD());
+
+    vm.startPrank(RISK_COUNCIL);
+    steward.updateGhoSupplyCap(newSupplyCap);
```

Comment:
How about a check (after clearing mock calls) that the new supply cap was set successfully?